src/components/Title/Title.tsx

```diff
@@ -9,7 +9,14 @@ interface TitleProps {
 
 const TitleHeader = styled.h1`
     text-align: center;
-    margin-top: 1em;
+    margin: 1em;
+
+    @media only screen and (max-width: 480px) {
+        font-size: 25px;
+    }
+    @media only screen and (max-width: 380px) {
```

Comment:
Same comment here, try to make it responsive using min/max and viewport/container units instead.

Reply:
Done